client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable-us.controller.js

```diff
@@ -0,0 +1,69 @@
+angular.module("App").controller("DedicatedCloudLicencesSplaEnableUSCtrl", class DedicatedCloudLicencesSplaEnableUSCtrl {
+
+    constructor ($stateParams, $scope, $state, OvhHttp, User, constants) {
+        if (constants.target !== "US") {
+            $state.go("^");
+        }
+        this.$state = $state;
+        this.OvhHttp = OvhHttp;
+        this.User = User;
+        this.serviceName = $stateParams.productId;
+        this.selectedOffer = null;
+    }
+
+    $onInit () {
+        this.loading = true;
+        return this.User.getUrlOf("express_order").then((url) => {
+            this.expressOrderUrl = url;
+        }).catch((err) => {
+            this.$scope.setMessage(this.$translate.instant("dedicatedCloud_tab_licences_active_spla_load_fail"), {
```

Comment:
if an error occurs in controller initialization, the modal will hide the message I think. You should close the modal or display the message into the modal.
Maybe you should use Alerter to remove the use of `$scope`?